dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/UserNotification.java

```diff
@@ -31,98 +32,117 @@ public class UserNotification implements Serializable, JpaEntity<Long> {
     @ManyToOne
     @JoinColumn(nullable = false)
     private AuthenticatedUser user;
+
     @ManyToOne
     @JoinColumn(nullable = true)
     private AuthenticatedUser requestor;
```

Comment:
Can we create a task to get rid of this and `additionalMessage`?

Reply:
https://github.com/CeON/dataverse/issues/2202